src/features/cookies.py

```diff
@@ -19,32 +25,23 @@ class Cookies(MetadataBase):
         "https://raw.githubusercontent.com/easylist/easylist/master/easylist_cookie/easylist_cookie_specific_hide.txt",
         "https://raw.githubusercontent.com/easylist/easylist/master/easylist_cookie/easylist_cookie_thirdparty.txt",
     ]
+
+    # fixme: still needed because it modifies the setup call...
```

Comment:
What would be the alternative?

Reply:
This gets resolved in the follow up PR, where the inheritance schema changes (there will be an `AdBlockBased` Base class). Changing to that base class will eliminate most of the enums that modulate how `MetadataBase` works.